src/rdb.c

```diff
@@ -3645,6 +3645,18 @@ void bgsaveCommand(client *c) {
     if (c->argc > 1) {
         if (c->argc == 2 && !strcasecmp(c->argv[1]->ptr, "schedule")) {
             schedule = 1;
+        } else if (c->argc == 2 && !strcasecmp(c->argv[1]->ptr, "cancel")) {
```

Comment:
nit: should we start using `strncasecmp`?

Reply:
I changed this to be a subcommand